docs/05-technical-reference/00-custom-resources/ac-01-application.md

```diff
@@ -36,34 +36,77 @@ spec:
-| **spec.services.entries.centralGatewayUrl** | No | Specifies the URL of Application Gateway. Internal address resolvable only within the cluster. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
-| **spec.services.entries.accessLabel** | No | Specifies the label used in Istio rules in Application Connector. This field is required for the API entry type. |
-| **spec.services.entries.targetUrl** |  No | Specifies the URL of a given API. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
-| **spec.services.entries.oauthUrl** | No | Specifies the URL used to authorize with a given API. This field is required for the API entry type. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
-| **spec.services.entries.credentialsSecretName** | No | Specifies the name of the Secret which allows you to call a given API. This field is required if **spec.services.entries.oauthUrl** is specified. In the [Compass mode](../../01-overview/main-areas/application-connectivity/README.md), it's provided by Runtime Agent. In the [standalone mode](../../01-overview/main-areas/application-connectivity/README.md), you have to provide it yourself. |
+<!-- The table below was generated automatically -->
+<!-- Some special tags (html comments) are at the end of lines due to markdown requirements. -->
+<!-- The content between "TABLE-START" and "TABLE-END" will be replaced -->
+<!-- TABLE-START -->
+<!-- Application v1alpha1 applicationconnector.kyma-project.io -->
+| Parameter         | Description                                   |
+| ---------------------------------------- | ---------|
+| **spec.accessLabel** |  |
+| **spec.compassMetadata** |  |
+| **spec.compassMetadata.applicationId** |  |
+| **spec.compassMetadata.authentication** |  |
+| **spec.compassMetadata.authentication.clientIds** |  |
+| **spec.description** | Describes the connected Application. |
+| **spec.displayName** |  |
+| **spec.encodeUrl** | Allows for URL encoding. If set to `false`, your URL segments stay intact. |
+| **spec.group** |  |
+| **spec.labels** | Defines the labels of the Application. |
+| **spec.longDescription** |  |
+| **spec.providerDisplayName** |  |
```

Comment:
`providerDisplayName` is still set by the Compass Runtime Agent, but I suppose it could be removed. Needs some short investigation.